data/quarg/quarg missions.txt

```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
 			label end
 			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
 				decline
+
+
+
+mission "Quarg Pug Arfecta Warning"
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "ringworld"
+	on offer
+		conversation
+			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
+			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
+			choice
+				`	"It's a long story..."`
+					goto evasive
+				`	"I prefer to frame it as an act of righteous defiance."`
+					goto righteous
+				`	"The Pug fancy themselves gods, but God is dead, soo..."`
+					goto anarchy
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	"It matters not how it happened."`
+				goto end
+			label righteous
+			`	“You fancy yourself a savior or a revolutionary then?” the Quarg says with what you assume is a sarcastic tone.`
+				goto end
+			label anarchy
+			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`
+				goto end
+			label greedy
+			`	"A baby in swaddling, as I said before."`
+				goto end
+			label end
+			`	"We give you a warning. You know not what you have child, the weapon you wield."`
+			`	"We have seen its power. We have felt its power. We have learned from its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you direct this power, and not it you."`
+			`	"And be careful to remember that we have learned to overcome it."`
+			`	The Quarg then curtly motions farewell and leaves you to your business. As you go you feel the crowd's glare follow you.`
```

Comment:
they motion farewell? would they really do that, in this context?

Reply:
Like a dismissive hand motion from the center down and out. Like see ya and a get out of here motion

Reply:
Ideas welcome most definitely welcome here.

Reply:
Maybe they just suddenly turn back, as a way of saying this conversation is over?
All Quargs could turn around Klingon style but that'd be a bit too much, though just one could be nice

Reply:
@Arachi-Lover Thoughts?

Reply:
You could have the Quarg clearly signal the end of the conversation with the dismissive motion or such, and then explicitally take one more, long look at the Arfecta. The crowd can then be said to disperse in a few minutes, but a few Quarg remain, gawking at the player character and/or the Arfecta.

Reply:
Updated this line. I think the current version addresses all feedback.